setup.py

```diff
@@ -20,6 +20,7 @@
     author='Ryan Tilder',
     author_email="service-dev@mozilla.com",
     url="http://mozilla.org",
+    install_requires=["M2Crypto"],
```

Comment:
We probably integrate this library using ``pip install ... --no-deps`` so it is most likely ok.